website/addons/owncloud/tests/utils.py

```diff
@@ -0,0 +1,61 @@
+import mock
+
+from owncloud import Client as OwnCloudClient
+from owncloud import FileInfo
+
+from tests.factories import ExternalAccountFactory
+from website.addons.base.testing import OAuthAddonTestCaseMixin, AddonTestCase
+from website.addons.owncloud.model import OwnCloudProvider
+from website.addons.owncloud.model import AddonOwnCloudNodeSettings
+from website.addons.owncloud.tests.factories import (
+    OwnCloudAccountFactory, OwnCloudNodeSettingsFactory,
+    OwnCloudUserSettingsFactory
+)
+
+class OwnCloudAddonTestCase(OAuthAddonTestCaseMixin, AddonTestCase):
+
+    short_name = 'owncloud'
+    full_name = 'OwnCloud'
+    ADDON_SHORT_NAME = 'owncloud'
+    ExternalAccountFactory = OwnCloudAccountFactory
+    Provider = OwnCloudProvider
+    NodeSettingsFactory = OwnCloudNodeSettingsFactory
+    NodeSettingsClass = AddonOwnCloudNodeSettings
+    UserSettingsFactory = OwnCloudUserSettingsFactory
+
+    def set_node_settings(self, settings):
+        super(OwnCloudAddonTestCase, self).set_node_settings(settings)
+        settings.folder_id='/Documents'
+        settings.external_account = self.external_account
```

Comment:
The other test classes should be handling this if necessary, please remove this line.